domain.yml

```
@@ -369,7 +372,7 @@ templates:
   utter_explain_nlu:
     - text: 'Rasa NLU performs Natural Language Understanding, which means taking free-form text like "Please send the confirmation to amy@example.com" and turning it into structured data.'
   utter_explain_core:
-    - text: "Rasa Core performs dialogue management, which means keeping track of a conversation and deciding how to proceed."
+    - text: "Rasa Core performs Dialog Management, which means keeping track of a conversation, and deciding how to proceed."
```

```suggestion
    - text: "Rasa Core performs dialogue management, which means keeping track of a conversation and deciding how to proceed."
```
Sorry i'd keep this the way it is :D